store/package.json

```diff
@@ -1,26 +1,17 @@
 {
   "name": "instanssi-store",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Instanssi.org store",
+  "type": "module",
   "scripts": {
-    "test": "cross-env NODE_ENV=test jest",
-    "build": "cross-env NODE_ENV=production webpack",
-    "watch": "cross-env NODE_ENV=development webpack --watch"
+    "build": "vite build",
+    "dev": "vite",
+    "test": "vitest"
   },
   "author": "Tommi Teistelä <totateis@gmail.com>",
   "license": "MIT",
-  "dependencies": {
-    "core-js": "~2.5.3"
-  },
   "devDependencies": {
-    "babel-core": "~6.26.0",
-    "babel-jest": "~22.1.0",
-    "babel-loader": "~7.1.2",
-    "babel-preset-es2015": "~6.24.1",
-    "cross-env": "~5.1.3",
-    "html-loader": "~0.5.5",
-    "jest": "~22.1.4",
-    "raw-loader": "~0.5.1",
-    "webpack": "~3.10.0"
+    "@types/node": "^22.0.0",
+    "vite": "~7.2.7"
```

Comment:
vitest too ?

Reply:
Oh, it worked. Guess this can be merged.